extensions/json-language-features/server/src/jsonServer.ts

```diff
@@ -424,6 +425,21 @@ export function startServer(connection: Connection, runtime: RuntimeEnvironment)
 		}, [], `Error while computing document symbols for ${documentSymbolParams.textDocument.uri}`, token);
 	});
 
+	connection.onCodeAction((codeActionParams, token) => {
+		return runSafeAsync(runtime, async () => {
+			const document = documents.get(codeActionParams.textDocument.uri);
+			if (document) {
+				const sortCodeAction = CodeAction.create('Sort JSON', CodeActionKind.Source.concat('.sort', '.json'));
+				sortCodeAction.command = {
+					command: 'json.sort',
+					title: 'Sort JSON'
```

Comment:
This needs to made translatable 
(
```suggestion
					title: l10n.t('Sort JSON')
````

Reply:
Also add `import * as l10n from '@vscode/l10n';`